synfig-studio/src/synfigapp/canvasinterface.cpp

```diff
@@ -985,74 +998,98 @@ CanvasInterface::import_sequence(
 			}
 			
 			String short_filename = CanvasFileNaming::make_short_filename(get_canvas()->get_file_name(), filename);
-			
+
 			try {
 				layer = add_layer_to("Import",get_canvas());
 				int w, h;
 				if (!layer)
 					throw int();
 				if (!layer->set_param("filename", ValueBase(short_filename)))
 					throw int();
+
+
+				curr_layer= modules::lyr_std::Import::Handle::cast_dynamic(layer);
```

Comment:
every `dynamic_cast` (or cast_dynamic) should be followed by check for null instance. A cast failure is possible (if layer is not of Import type.

Reply:
Okay! Will do!